test/ecdsatest.c

```diff
@@ -103,7 +103,7 @@ static int x9_62_test_internal(int nid, const char *r_in, const char *s_in)
 {
     int ret = 0;
     const char message[] = "abc";
-    unsigned char digest[20];
+    unsigned char digest[SHA_DIGEST_LENGTH];
```

Comment:
Would it make sense for this to be `EVP_MAX_MD_SIZE` instead?

It isn't important at the moment, but it is future safe.

Reply:
In this case digest must be sha1. In next case i used SHA512_DIGEST_LENGTH that is equal to EVP_MAX_MD_SIZE.

Reply:
Okay.  That's reasonable.  BTW: SHA512_DIGEST_LENGTH is equal to EVP_MAX_MD_SIZE _at the moment_.  This could and likely will change in future.